src/other.c

```diff
@@ -260,6 +260,11 @@ static const char* igraph_i_plfit_error_message = 0;
 
 static void igraph_i_plfit_error_handler_store(const char *reason, const char *file,
         int line, int plfit_errno) {
+
+    IGRAPH_UNUSED(file);
+    IGRAPH_UNUSED(line);
+    IGRAPH_UNUSED(plfit_errno);
```

Comment:
In cases when the arguments are unused but must be there because they are part of a callback signature, I think an easier solution is just to remove the _name_ of the argument and leave its type there. Can you test whether it would work with MSVC? If so, let's do it this way because it improves readability.

Reply:
Unfortunately that does not work. Or well, that is, it gives a different warning: `error C2055: expected formal parameter list, not a type list`. According to the Microsoft docs, this is also expected by ANSI C, so this is then a warning that will presumably also pop up with other compilers. So, probably best to keep this as is.